### PR TITLE
[To rel/0.13] [IOTDB-3624] Meet error when inserting data in SQL without timestamp

### PR DESCRIPTION
**Pull request by lyssom**
## Description
fix bug
https://issues.apache.org/jira/browse/IOTDB-3624
cherry pick with https://github.com/apache/iotdb/pull/6642

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java [hunk 1]
```diff
@@ -105,6 +105,17 @@ public void testInsertWithoutTime() throws SQLException, InterruptedException {
     st1.close();
   }
 
+  @Test
+  public void testInserthaventTime() throws SQLException, InterruptedException {
```
Comment (liuminghui233) [inline integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java:109]: I can't understand the method name, is this a typo?

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java [hunk 2]
```diff
@@ -105,6 +105,17 @@ public void testInsertWithoutTime() throws SQLException, InterruptedException {
...
+    Statement st0 = connection.createStatement();
+    st0.execute("insert into root.t1.wf01.wt01(status, temperature) values (1, 11)");
+
+    ResultSet rs1 = st0.executeQuery("select count(status) from root.t1.wf01.wt01");
+    rs1.next();
+    long countStatus = rs1.getLong(1);
+    System.out.println(countStatus);
```
Comment (liuminghui233) [inline integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java:116]:
```suggestion
    Assert.assertEquals(countStatus, 1L);
```

**Review by liuminghui233 — CHANGES_REQUESTED** [review]

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java (hunk 1 above)
Comment (lyssom) [reply integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java:109]: I think  have not = haven’t = Without. I will change it.

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java (hunk 2 above)
Comment (lyssom) [reply integration/src/test/java/org/apache/iotdb/db/integration/IoTDBInsertWithoutTimeIT.java:116]: ok

**Comment by liuminghui233** [comment]
IoTDBInsertWithoutTimeIT  is failed. PTAL :-)

**Pull request merged**